app/main.py

```diff
@@ -1 +1,39 @@
-# write your code here
+class Animal:
+    alive = list()
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        if health > 0:
+            Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        object_items_str = ", ".join(
+            (f"{k.capitalize()}: {v}" for k, v in vars(self).items())
+        )
+        return f"{{{object_items_str}}}"
```

Comment:
Write `__repr__` method using one f-string and defining `Name`, `Health`, and `Hidden` directly without using additional cycles. It would help to improve the readability.

Reply:
It wont pass tests, its ok?